src/AppRouter.js

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import HomePageBoilerplate from './pages/HomePageBoilerplate';
 import HomePage from './pages/HomePage';
 import ExamplePage from './pages/ExamplePage';
 import OtherPage from './pages/OtherPage';
 import NotFound from './pages/NotFound';
 
-
 class AppRouter extends Component {
     render(){
         return (
```

Comment:
Is the React.Fragment still necessary or left-over?

Reply:
I think it can be removed

Reply:
Blow it up! 💣💥